content/guides/component-testing/custom-mount-angular.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Custom Mount Commands for Angular
+---
+
+In some instances you may find yourself doing repetitive work during mounting.
+In order to reduce boilerplate you may find it useful to create a custom mount
+command.
+
+Here is an example of defaulting `autoSpyOutputs` for every mounted component:
+
+### autoSpyOutputs
+
+<code-group>
+<code-block label="support/component.ts" active>
+
+```ts
+declare global {
+  namespace Cypress {
+    interface Chainable {
+      mount<T>(
+        component: Type<T> | string,
+        config?: MountConfig<T>
+      ): Chainable<MountResponse<T>>
+    }
+  }
+}
```

Comment:
```suggestion
declare global {
  namespace Cypress {
    interface Chainable {
      mount: typeof mount
    }
  }
}
```

I'd stick with the one we scaffold so it doesn't confuse the user if they are following along and this differs